src/test/java/org/broadinstitute/hellbender/tools/funcotator/mafOutput/MafOutputRendererUnitTest.java

```diff
@@ -123,7 +123,8 @@ private MafOutputRenderer createMafOutputRenderer(final File outputFile, final S
                 FuncotatorArgumentDefinitions.LOOKAHEAD_CACHE_IN_BP_DEFAULT_VALUE,
                 new FlankSettings(0, 0),
                 false,
-                FuncotatorUtils.DEFAULT_MIN_NUM_BASES_FOR_VALID_SEGMENT
+                FuncotatorUtils.DEFAULT_MIN_NUM_BASES_FOR_VALID_SEGMENT,
+                FuncotatorUtils.DEFAULT_SPLICE_SITE_WINDOW_SIZE
```

Comment:
Should there be a test that supplies a non-default value for this new argument?

Reply:
Yeah, that's probably a good idea, though I was hoping to avoid adding in more tests for this case.  I'll add one or two edge cases.

Reply:
If it's non-trivial, then we can live with the existing test coverage.

Reply:
No, it's fine.  It's the right thing to do.

Tests are in!